tests/test_rsa_keys.py

```diff
@@ -157,26 +158,30 @@ def test_verify_rsa_signature(self):
           scheme, public_rsa, data))
 
 
-  def test_verify_rsa_pss_sha256(self):
+  def test_verify_rsa_pss_different_salt_lengths(self):
     rsa_scheme = 'rsassa-pss-sha256'
-    data = 'The ancients say the longer the salt, the more provable the security'.encode('utf-8')
+    data = 'The ancients say, salt length does not matter that much'.encode('utf-8')
```

Comment:
Mathematically speaking, apparently this is not strictly true, although practically it might be the case 😉

Reply:
Yeah, I would also not trust every word the ancients say. ;)